image_client.py

```diff
@@ -78,14 +88,18 @@ def delete_from_image_server(self, attach_loc, collection):
             'token': self.generate_token(attach_loc),
         }
         url = self.build_url("filedelete")
-        logging.debug(f"Deleting {url} from server")
+        self.logger.debug(f"Deleting {url} from server")
         r = requests.post(url, data=data)
         if r.status_code == 404:
             raise FileNotFoundException
         if r.status_code != 200:
             print(f"Deletion failed, aborted: {r.status_code}:{r.text}")
             raise DeleteFailureException
 
```

Comment:
I think we no longer need an artifact of the update_iptc method in the image client as we've abandoned using iptc as a distinct standard from exif.